### ISSUE TITLE
Works fine, but not unattended (batch)

### ISSUE THREAD

**Issue by stg** [Metabolix/HackBGRT#203]
I've set up and have been usuing HackBGRT manually on multiple computers, and it works fine.

However, when I do it unattended/silent/batch mode from a script I get an error message during boot just before the new background image is displayed. It still works, but the error message is confusing to users.

The command lines I have tried are:

setup.exe batch install
setup.exe batch install allow-secure-boot
setup.exe batch install allow-secure-boot enable-overwrite enable-bcdedit enable-entry

All of these lead to the error message.

When I do a manual install (Press "I" at prompt, close Microsoft Paint when opened, press the "anykey" when done) I do not get the error message.

How can I do a batch install that does *exactly* what the manual install does (except prompt and open Paint, obviously)?

**Comment by Metabolix** [comment]
You need to attach the error and preferably the log file (`setup.exe batch show-boot-log` and the attach `setup.log`) to get any help with this.

**Comment by stg** [comment]
Thank you :)

Well, the question was "How can I do a batch install that does exactly what the manual install does", and I found the answer in the code of course. I feel a little ashamed to ask before properly checking. Sorry.

Hitting "I" obviously sets the program up to do `disable`, `install`, and `enable-bcdedit`.

So the equivalent command line would be `setup.exe batch disable install enable-bcdedit`.

This did indeed solve the problem.

My theory is that I created the problem for myself when I initially ran just `setup.exe batch install`, and since my subsequent commands did not include `disable` it never sorted itself out.
